src/providers/twitch/TwitchIrcServer.hpp

```diff
@@ -42,6 +44,7 @@ class TwitchIrcServer final : public AbstractIrcServer, public Singleton
     IndirectChannel watchingChannel;
 
     PubSub *pubsub;
+    SeventvEventApi *eventApi;
```

Comment:
Initialize to nullptr here directly (related to the comment in the related .cpp file) and/or swap this with a smart pointer anyway.

Reply:
> and/or swap this with a smart pointer anyway.

I agree. I've been following `PubSub`, which uses a c-style pointer.

Changed it to `unique_ptr`.